suse_migration_services/units/reboot.py

```diff
@@ -66,16 +66,13 @@ def main():
                         )
                     )
                 )
+            if not MigrationConfig().is_soft_reboot_requested():
+                command_list = ['systemctl', 'reboot']
+            else:
+                command_list = ['systemctl', 'kexec']
```

Comment:
All looks good, the only thing that is different for the `if` is the command to run
maybe something like this:

```python
            if not MigrationConfig().is_soft_reboot_requested():
                restart = 'reboot'
            else:
                restart = 'kexec'    
            command_list = ['systemctl', restart]
```